e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -62,3 +65,55 @@ const DATA_GRID_SELECTOR = '#container';
     await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR);
   });
 });
+
+test('Focused cell should not flick', async (t) => {
+  const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+  const firstCell = dataGrid.getDataCell(0, 0).element;
+  const secondCell = dataGrid.getDataCell(1, 0).element;
+  await t.click(firstCell);
+  await t.click(secondCell);
+
+  await ClientFunction(() => {
+    // @ts-expect-error
+    window.counter = 0;
```

Comment:
Let's avoid the ```@ts-expect-error``` in cases like this (add a custom ```window``` property)
It's better to create more wide type here, e.g:

```typescript
type WindowWithTestData = Window & { counter: number };
```

And use this type in ```ClientFunction```:
```typescript
const windowWithTestData = window as WindowWithTestData;
windowWithTestData.counter = 0;
```